stacker/providers/aws/default.py

```diff
@@ -106,18 +111,43 @@ def cloudformation(self):
 
         return self._cloudformation
 
+    @property
+    def listener(self):
+        # deals w/ multiprocessing issues w/ sharing ssl conns
+        # see https://github.com/remind101/stacker/issues/196
+        pid = os.getpid()
+        if pid != self._pid or not self._listener:
+            session = get_session(self.region)
+            self._listener = CloudListener(
+                LISTENER_NAME, session=session)
+            self._listener.start()
+
+        return self._listener
+
+    def pre_run(self):
```

Comment:
Lets call this `flush_events` or something similar to better represent what it's doing.